src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_uncompyle.py

```diff
@@ -0,0 +1,642 @@
+"""
```

Comment:
Just to note, as commented, this was a bit of a proof of concept to see how much work it'd be to make a decompiler (so, I worked a bit more than a day just to see what'd be the size of the task if we wanted to do it given that I haven't found any non-GPL implementation -- after this experiment, I think it'd be a really big task -- I think it's probably close to 1-2 months to implement something really usable so, I'm unsure whether we want to tackle it or not in the future).

Reply:
Given the 1-2 month duration, I think we should put this on hold.

Reply:
Sure, agreed on the full decompile, still, this comment is only related to that particular file (the current pull request integrates a version that shows a simple bytecode decompile which should be functional).

Reply:
Users are often confused by "Source not found" error message, and believe that it applies to debug session as a whole - i.e. they don't realize that they can still continue stepping or F5 to keep going. So I think this is worthwhile even with bytecode alone.

Reply:
Agreed... I'll rename it.